pages/lessons/index.tsx

```diff
@@ -1,19 +1,26 @@
 import Head from 'next/head'
-import Link from 'next/link'
-import { Heading, Flex, Stack, Divider } from '@chakra-ui/react'
+import { Flex, Stack, Heading } from '@chakra-ui/react'
 import fs from 'fs'
 import path from 'path'
 import matter from 'gray-matter'
 import { ContentBanner } from '../../components/ContentBanner'
 
 interface LessonProps {
   lessons: {
+    path: string
```

Comment:
Changing what's passed-in made me think of another potential approach. What if we just add front-matter that has the appropriate section for all content.

e.g.

section: Projects  (and eventually this could be "Track 1: NFT" when we start adding more tracks)
section: Fundamentals

Then the file processing code could read files in any hierarchy (which is friendly for content authors) but which section the content falls under depends on front-matter.